client/src/components/layout/Navbar.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { authActions } from '../../sagas/actions';
+
+const Navbar = ({ currentUser, logout }) => {
+  return (
+    <nav className='h-12 px-4 bg-blue-900 flex justify-between items-center text-gray-400'>
+      <Link
+        to={currentUser.loggedIn ? '/home' : '/'}
+        className='uppercase text-xl tracking-wider hover:text-white'
+      >
+        Cribbage
+      </Link>
+      {!currentUser.loggedIn ? (
+        <div className='flex'>
+          <Link to='/login' className='px-2 hover:text-white'>
+            Login
+          </Link>
+          <Link to='/' className='px-2 hover:text-white'>
+            Register
+          </Link>
+        </div>
+      ) : (
+        <button
+          onClick={() => logout()}
+          className='focus:outline-none hover:text-white'
+        >
+          Logout
+        </button>
+      )}
```

Comment:
This "top bar" feels like it _could_ be broken down further, but until we need that, I'm chill with it as is.

![Screen Shot 2020-06-09 at 11 09 57 PM](https://user-images.githubusercontent.com/23204038/84226043-5e344b80-aaa6-11ea-8ca4-3a3f1919e85b.png)

![Screen Shot 2020-06-09 at 11 10 13 PM](https://user-images.githubusercontent.com/23204038/84226059-67251d00-aaa6-11ea-9a76-f83f0a84c97f.png)

Reply:
Code-wise, or visually?

Reply:
In the code: I think it'd make logical sense to have more partitions rather than just letting it all live here. But this might be a YAGNI so it's chill as is.